### PR TITLE
net-p2p/rtorrent: version bump to 0.9.7

### PR DESCRIPTION
**Pull request by ss23**
Also bumps net-libs/libtorrent (dependency)

### CODE REVIEW

**Comment by ss23** [comment]
@a17r Fixed 👍

File: net-libs/libtorrent/libtorrent-0.13.7.ebuild [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+LICENSE="GPL-2"
+
+# The README says that the library ABI is not yet stable and dependencies on
+# the library should be an explicit, syncronized version until the library
+# has had more time to mature. Until it matures we should not include a soname
+# subslot.
+SLOT="0"
+
+KEYWORDS="~amd64 ~arm ~hppa ~ia64 ~ppc ~ppc64 ~sparc ~x86 ~x86-fbsd ~amd64-linux ~arm-linux ~x86-linux ~ppc-macos ~x64-macos ~x86-macos ~sparc-solaris ~x64-solaris"
+IUSE="debug libressl ssl test"
```
Comment (a17r) [inline net-libs/libtorrent/libtorrent-0.13.7.ebuild]: I see you removed ipv6 flag, what is that about?

Comment (ss23) [reply net-libs/libtorrent/libtorrent-0.13.7.ebuild]: The IPv6 use flag no longer had any effect. The ability to toggle IPv6 support at compile time was removed from the project. It's resulted in a bug referenced further down in the source: https://github.com/rakshasa/rtorrent/issues/732

Comment (a17r) [reply net-libs/libtorrent/libtorrent-0.13.7.ebuild]: this like that are good to have in the commit message, in case someone asks questions afterwards.

Comment (ss23) [reply net-libs/libtorrent/libtorrent-0.13.7.ebuild]: Pushed an amended commit message with further information

Comment (a17r) [reply net-libs/libtorrent/libtorrent-0.13.7.ebuild]: thanks!

**Pull request closed without merging**